tests/rados/test_scrub_log.py

```diff
@@ -126,18 +127,24 @@ def check_scrub_workflow(installer, rados_obj, acting_sets, task) -> bool:
     for pgid in acting_sets.keys():
         log.debug(f"Checking OSD logs of PG: {pgid}. OSDs : {acting_sets[pgid]}")
         init_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+        log.debug(f"Initial time when {task} was started : {init_time}")
 
         if task == "scrub":
-            rados_obj.run_scrub(pgid=pgid)
+            log.debug(f"Running scrub on pg : {pgid}")
+            if not rados_obj.start_check_scrub_complete(pg_id=pgid):
+                log.error(f"Could not complete scrub on pg : {pgid}")
+                return False
             log_lines = [f"{pgid} scrub starts", f"{pgid} scrub ok"]
         elif task == "deep-scrub":
-            rados_obj.run_deep_scrub(pgid=pgid)
+            if not rados_obj.start_check_deep_scrub_complete(pg_id=pgid):
+                log.error(f"Could not complete scrub on pg : {pgid}")
```

Comment:
Please modify logger to "Could not complete deep scrub on pg"

Reply:
Updated the logging.